azure-pipelines.yaml

```diff
@@ -27,3 +27,23 @@ stages:
           solution: VectorRumble.Desktop/VectorRumble.Desktop.csproj
           configuration: Release
           msbuildArguments: /restore /v:diag
+    - job: BuildUWP
+      displayName: UWP
+      pool:
+        vmImage: vs2017-win2016
+      steps:
+      - bash: |
+          curl -o MonoGameSetup.exe -L https://github.com/MonoGame/MonoGame/releases/download/v3.7.1/MonoGameSetup.exe
+      - script: |
+          dir
+          MonoGameSetup.exe /S
+      - task: NuGetCommand@2
+        inputs:
+          command: 'restore'
+          restoreSolution: 'VectorRumble.sln'
+      - task: MSBuild@1
+        displayName: 'Build solution VectorRumble.UWP/VectorRumble.UWP.csproj'
+        inputs:
+          solution: VectorRumble.UWP/VectorRumble.UWP.csproj
+          configuration: Release
+          msbuildArguments: /restore /v:diag /p:AppxPackage=false
```

Comment:
If we don't force `ApxPackage=false` then we get a file missing error for `Package.StoreAssociation.xml `

Reply:
Ok, so We need to add that with the Windows Store. I think I'll need to do that since it will be released under infspace studios.